Dockerfile.build

```diff
@@ -15,5 +19,10 @@ RUN rustup target add wasm32-unknown-unknown
 RUN cargo install webassembly-test-runner
 RUN cargo install cargo-llvm-cov
 
+RUN pacman -S python python-pip --noconfirm --disable-download-timeout
+RUN curl -sSL https://install.python-poetry.org | python3 -
+
+RUN pacman -S allure --noconfirm
```

Comment:
Join multiple `RUN`s in one `RUN` command for pacman packages to reduce the image layers.

Reply:
did it